coderd/database/queries/insights.sql

```diff
@@ -22,18 +22,18 @@ GROUP BY workspace_agent_stats.user_id, users.username, users.avatar_url
 ORDER BY user_id ASC;
 
 -- name: GetTemplateInsights :one
--- GetTemplateInsights has a granularity of 5 minutes where if a session/app was
--- in use, we will add 5 minutes to the total usage for that session (per user).
-WITH d AS (
-	-- Subtract 1 second from end_time to avoid including the next interval in the results.
-	SELECT generate_series(@start_time::timestamptz, (@end_time::timestamptz) - '1 second'::interval, '5 minute'::interval) AS d
-), ts AS (
+-- GetTemplateInsights has a granularity of 1 minute where if a session/app was
+-- in use during a minute, we will add 1 minute to the total usage for that
+-- session/app (per user).
```

Comment:
Can we extract this into a separate PR? If we introduce a bug, it will be easier to spot the issue (possibly related to changed interval than app insights).

Reply:
I'll revert the 1m -> 5m change and I can work on query optimizations in another PR (optimization hasn't been a priority yet).